en/docs/reference/connectors/ISO8583-connector/ISO8583-connector-example.md

```diff
@@ -18,7 +18,7 @@ If you do not want to configure this yourself, you can simply [get the project](
 
 Follow these steps to set up the Integration Project and the Connector Exporter Project.
 
-{!references/connectors/importing-connector-to-integration-studio.md!}
+{!reference/connectors/importing-connector-to-integration-studio.md!}
```

Comment:
Shall we remove the .md extensions from all the files?

Reply:
Fixed.

Reply:
Actually we can't remove it from this particular one. The links break without .md as these are files getting pulled into the page and it doesn't work without the extension.

Reply:
ack